frontend/portals/HistorySuggestions/index.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { I18n } from '@shopgate/engage/components';
+import { themeConfig } from '@shopgate/pwa-common/helpers/config';
+import Button from '@shopgate/pwa-ui-shared/Button';
+import { isIOSTheme } from '@shopgate-ps/pwa-extension-kit/env/helpers';
+import { css } from 'glamor';
+import classnames from 'classnames';
+import connect from './connector';
+import SuggestionList from './components/SuggestionList';
+import stylesFile from './style';
+
+const { colors } = themeConfig;
+const isIOS = isIOSTheme();
+
+const styles = {
+  deleteHistory: isPersistentSearchBar => css({
+    textDecoration: 'underline',
+    marginLeft: isIOS || isPersistentSearchBar ? 46 : 72,
+    marginTop: 10,
+    color: colors.shade3,
+    fontSize: 14,
+  }).toString(),
+};
+
+/**
+ * @return {JSX}
+ */
+const HistorySuggestions = ({
+  onClick,
+  searchHistory,
+  suggestions,
+  searchPhrase,
+  children,
+  deleteSearchHistory,
+  visible,
+  bottomHeight,
+  name,
+}) => {
+  const isPersistentSearchBar = name === 'persistent-search-bar.search.suggestions';
+  if (!visible || (suggestions && suggestions.length && searchPhrase !== '') || !searchHistory.length) {
```

Comment:
I guess the `if (!visible || searchPhrase !== '' || !searchHistory.length)` would be sufficient (not show history when user already typed something, otherwise history does not much to search query)
![image](https://user-images.githubusercontent.com/18292310/88527656-db3b6600-cffd-11ea-9c67-98801bd00e17.png)

Reply:
And maybe its a reason why its not shown on home page for persistent search bar and empty query